lib/project_types/script/messages/messages.rb

```diff
@@ -130,11 +124,6 @@ module Messages
           Turn on script in development store.
             Usage: {{command:%s enable}}
           HELP
-          extended_help: <<~HELP,
```

Comment:
Same here, needed by this change: https://github.com/Shopify/shopify-app-cli/pull/727